emcc.py

```diff
@@ -4188,7 +4200,8 @@ def process_libraries(state, linker_inputs):
     lib = strip_prefix(flag, '-l')
 
     logger.debug('looking for library "%s"', lib)
-    js_libs, native_lib = building.map_to_js_libs(lib)
+
+    js_libs, native_lib = building.map_to_js_libs(lib, options.embind_emit_tsd)
```

Comment:
Can we just pass `options.embind_emit_tsd` though instead of the whole options object?